NWNXLib/Services/Events/Events.cpp

```diff
@@ -138,9 +138,8 @@ void EventsProxy::ClearEvent(const std::string& eventName)
         throw std::runtime_error("Tried to clear unrecognised event.");
     }
 
-    Events::RegistrationToken concreteToken = std::move(*token);
+    m_proxyBase.ClearEvent(*token);
```

Comment:
I have a vague recollection that these had to be done in the old order, but I don't remember why at all, and looking at the code now it doesn't seem like it. *shrug*

Did you run the unit test suite with these changes?

Reply:
I'll just drop this commit for now.  I really don't understand enough about the code base yet.

Reply:
I don't think `ClearEvent()` is actually used anywhere right now.